check_http.go

```diff
@@ -41,20 +43,22 @@ func (c CheckHTTP) Execute(timeout time.Duration) error {
 
 	req, err := http.NewRequest(c.Method, c.Path, nil)
 	if err != nil {
-		return err
+		return false, fmt.Errorf("building http request failed %s", err)
 	}
 
 	res, err := client.Do(req)
 	if err != nil {
-		return err
+		log.Println(log.LevelInfo, "HTTP request failed: %s", err)
```

Comment:
Yeah I was concerned about this. The reason I did it was because these result in a `false` result rather than an error now, so the actual error message gets swallowed. The only way I can see of solving this is to have a more expressive return type that also associated a "reason" with a false result.